package.json

```diff
@@ -36,8 +36,8 @@
     "@angular/router": "~12.0.2",
     "@fortawesome/fontawesome-free": "^5.13.0",
     "ng-event-source": "^1.0.14",
-    "primeicons": "^2.0.0",
-    "primeng": "^11.3.1",
+    "primeicons": "^4.1.0",
+    "primeng": "^12.0.0-rc.1",
```

Comment:
Is it safe to use a release candidate?

Reply:
Probably no, but they don't have a non-rc release yet for Angular 12.